compiler/il/OMRILOps.hpp

```diff
@@ -39,6 +39,8 @@ enum VectorOperation
    {
    vadd,
    vfma,
+   vabs,
+   vsqrt,
```

Comment:
I imagine this list will grow significantly.  Can it be ordered alphabetically for easier human readability?